crates/factor-outbound-pg/tests/factor_test.rs

```diff
@@ -1,39 +1,51 @@
-use anyhow::bail;
+use anyhow::{bail, Result};
 use spin_factor_outbound_networking::OutboundNetworkingFactor;
+use spin_factor_outbound_pg::client::Client;
 use spin_factor_outbound_pg::OutboundPgFactor;
 use spin_factor_variables::{StaticVariables, VariablesFactor};
 use spin_factor_wasi::{DummyFilesMounter, WasiFactor};
 use spin_factors::{anyhow, RuntimeFactors};
 use spin_factors_test::{toml, TestEnvironment};
+use spin_world::async_trait;
 use spin_world::v2::postgres::HostConnection;
+use spin_world::v2::postgres::{self as v2};
 use spin_world::v2::rdbms_types::Error as PgError;
+use spin_world::v2::rdbms_types::{ParameterValue, RowSet};
 
 #[derive(RuntimeFactors)]
 struct TestFactors {
     wasi: WasiFactor,
     variables: VariablesFactor,
     networking: OutboundNetworkingFactor,
-    pg: OutboundPgFactor,
+    pg: OutboundPgFactor<MockClient>,
+}
+
+fn factors() -> Result<TestFactors> {
+    let mut f = TestFactors {
+        wasi: WasiFactor::new(DummyFilesMounter),
```

Comment:
FYI this shouldn't be necessary any more; `OutboundNetworkingFactor`'s dependency on it is now optional.